confdb/confdb.go

```diff
@@ -1752,17 +1714,15 @@ func (p viewRule) isWriteable() bool {
 // pattern is an individual subkey of a dot-separated name or path pattern. It
 // can be a literal value of a placeholder delineated by curly brackets.
 type requestMatcher interface {
-	accessor
-
-	match(subkey accessor, matched *matchedPlaceholders) bool
-}
+	Accessor
 
-type writeOpts struct {
-	topLevel bool
+	match(subkey Accessor, matched *matchedPlaceholders) bool
 }
 
 type storageWriter interface {
-	write(sb *strings.Builder, matched *matchedPlaceholders, opts writeOpts)
+	Accessor
+
+	write(matched *matchedPlaceholders) Accessor
```

Comment:
could you please add a doc comment on what this does? I am slightly confused by the no-op implementations below